app/services/apply_mods_metadata.rb

```diff
@@ -93,39 +82,18 @@ def update_metadata
   end
 
   # Open a new version for the given object.
-  def commit_new_version
+  def open_version
     @cocina = VersionService.open(druid: cocina.externalIdentifier,
                                   description: "Descriptive metadata upload from #{original_filename}",
                                   opening_user_name: ability.current_user.sunetid)
   end
 
-  # Returns true if the given object is accessioned, false otherwise.
-  def accessioned?
-    (6..8).cover?(status)
-  end
-
-  # Checks whether or not a DOR object is in accessioning or not.
-  #
-  # @return [Boolean] true if the object is currently being accessioned, false otherwise
-  def in_accessioning?
-    (2..5).cover?(status)
-  end
-
-  # Checks whether or not a DOR object's status is OK for a descMetadata update. Basically, the only times we are
-  # not OK to update is if the object is currently being accessioned and if the object has status unknown.
-  #
-  # @return [Boolean] true if the object's status allows us to update the descripitive metadata, false otherwise
-  def status_ok?
-    [1, 6, 7, 8, 9].include?(status)
-  end
+  # Item must be open or openable? to be updated
+  def updatable?
+    return true if VersionService.open?(druid: cocina.externalIdentifier) || VersionService.openable?(druid: cocina.externalIdentifier)
```

Comment:
Using the class methods will result in separate calls to DSA for .open? and .openable? If you instantiate a VersionService and call the instance methods it will be only a single call to DSA.

Reply:
Thanks for the advice, will implement that.